tsdef/types.d.ts

```diff
@@ -164,8 +164,17 @@ export interface CreateLocalTrackOptions extends MediaTrackConstraints {
   workaroundWebKitBug1208516?: boolean;
 }
 
+export interface NoiseCancellationOptions {
+  sdkAssetsPath: string;
+  sdkFile: string;
+}
+
+export interface LocalAudioTrackOptions extends CreateLocalTrackOptions {
```

Comment:
Same here, we should export this in index.d.ts

Reply:
can you clarify -  I am not sure what need to change.

Reply:
I should be able to do this
```ts
import { LocalAudioTrackOptions } from 'twilio-video';

const options: LocalAudioTrackOptions = {foo};
```
But in order to do this, we need to export LocalAudioTrackOptions from tsdef/index.d.ts. Currently, I don't see it there.
By the way, were already doing the same pattern for `CreateLocalTrackOptions`. See this [line](https://github.com/twilio/twilio-video.js/blob/audio_processor/tsdef/index.d.ts#L54).

Reply:
got it! I will update, thanks, I do have a test app, that uses these, but it did not initialize these types specifically, so I missed this part.